features/provider/active_docs.feature

```diff
@@ -0,0 +1,32 @@
+Feature: ActiveDocs pages
+  As a provider
+  I want to manage my ActiveDocs
+
+  Background:
+    Given a provider is logged in
+      And the provider has 1 active doc
+
+  Scenario: Index renders with the proper elements
+    When I go to the provider active docs page
+    Then I should see "ActiveDocs" within the main title
+     And the table should contain the API
+
+  Scenario: Preview renders with the proper elements
+    When I go to the preview active docs page
+    Then I should see "ActiveDocs" within the main title
+
+  Scenario: Edit and the redirect after failure render with the proper elements
+    When I go to the edit active docs page
+    Then I should see "ActiveDocs" within the main title
+     And the service selector is in the form
```

Comment:
I would *use* the service selector instead of asserting it is in the form.

```
When I select a service from the service selector
And I save the form
Then the api doc spec is saved with this service linked
```

Reply:
@hallelujah Good 👍 In the cases in which we don't want to allow the selection of the service in the form, what should I do? Right now those cases have `And the service selector is not in the form`

Reply:
I think this is a valid case, but I'd put it into the a case of edit when apidoc spec is already attached with a service